app/models/concerns/git_hub_repoable.rb

```diff
@@ -42,8 +42,8 @@ def destroy_github_repository
   def push_starter_code
     return true unless starter_code_repo_id
 
-    repository              = GitHubRepository.new(organization.github_client, github_repo_id)
-    starter_code_repository = GitHubRepository.new(organization.github_client, starter_code_repo_id)
+    repository              = GitHubRepository.new(creator.github_client, github_repo_id)
+    starter_code_repository = GitHubRepository.new(creator.github_client, starter_code_repo_id)
```

Comment:
This is to make sure that the owner of the repository is the token that we are using, just in case they use a personal private repo.